lib-ee/emqx_ee_connector/src/emqx_ee_connector_influxdb.erl

```diff
@@ -139,6 +140,19 @@ on_get_status(_InstId, #{client := Client}) ->
 %% schema
 namespace() -> connector_influxdb.
 
+roots() ->
+    [
+        {config, #{
+            type => hoconsc:union(
+                [
+                    hoconsc:ref(?MODULE, influxdb_udp),
```

Comment:
This can be removed.  IIRC, we don't support UDP in 5.0 (unless that changed in the past 3 weeks).

Reply:
Ok, I have now removed all mentions of UDP in schemas.